static/js/controllers/reports.js

```diff
@@ -131,34 +188,65 @@ var _ = require('underscore'),
           return DB.get()
             .get(report)
             .then(FormatDataRecord);
-        } else {
-          return FormatDataRecord(report);
         }
+        return FormatDataRecord(report);
       };
 
       $scope.refreshReportSilently = function(report) {
-        _fetchFormattedReport(report)
+        return _fetchFormattedReport(report)
           .then(function(doc) {
-              _setSelected(doc[0]);
-            })
+            _setSelected(doc[0]);
+          })
           .catch(function(err) {
             $log.error('Error fetching formatted report', err);
           });
       };
 
+      var spliceSelected = function(id) {
```

Comment:
Heh. Yeah. There's inconsistency here. Initially I was using `_` to indicate private or internal functions, but then I realised that anything that's not defined on `$scope` is private or internal so it was unnecessary and confusing. I'll remove `_` from all functions in this controller.